src/hearings/pipes/camel-to-sentencecase.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+  name: 'camelToSentenceCase'
+})
+export class CamelToSentenceCasePipe implements PipeTransform {
+
+  constructor() {}
+
+  public transform(value: any): string {
+    return this.camelCaseToSentenceCase(value);
+  }
+
+  private camelCaseToSentenceCase(input: string): string {
+    if (!input) return '';
+    const split = input.replace(/([A-Z])/g, ` $1`).trim();
```

Comment:
I assume this is handling 'videoFacility'? If yes no need to do this as this is reference data we expect what referece data we got fom API is in correct format.